backend/app/main.py

```
@@ -60,161 +60,125 @@ async def lifespan(app: FastAPI):
-                        raise
-                    except Exception as e:
-                        logger.error(
-                            f"✗ Unexpected error running Alembic migrations: {e}"
-                        )
-                        raise
-                elif settings.ENVIRONMENT == "production":
-                    logger.warning(
-                        "Running in production mode. Database migrations must be run manually. "
-                        "Please execute 'alembic upgrade head' to apply pending migrations."
+                    # Run Alembic as subprocess to avoid output buffering issues
+                    result = subprocess.run(
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)